python/tank/deploy/tank_commands/install.py

```diff
@@ -366,6 +427,13 @@ def run_interactive(self, log, args):
             log.info("For example, to install the houdini engine into the Asset environment:")
             log.info("> tank install_engine Asset tk-houdini")
             log.info("")
+            log.info("Comment and structure preserving mode")
```

Comment:
Any chance you can refactor this? It's been copy pasted 3 times now and already there are discrepancies between the 3.